filebrowser/decorators.py

```diff
@@ -4,7 +4,10 @@
 
 from django.contrib import messages
 from django.core.exceptions import ImproperlyConfigured
-from django.core.urlresolvers import reverse
+try:
```

Comment:
The master branch supports Django 1.11+ which has `django.urls` so the try/except isn't needed.